### PR TITLE
Deploy on Docker

### PR DESCRIPTION
**Pull request by FallenDoll**
### TL;DR
Added a Docker deployment method to the project by creating a Dockerfile and a docker-compose.json file.

### Description
Added a Docker deployment method to the project, by writing dockerfile and docker-compose.json files. This way, it is convenient to run and test the project in different environments, and it also improves the project's portability and security.

### Changes
- Added Dockerfile
- Added docker-compose.json
- Modified src/database/database.py to to retrieve the MongoDB connection string from environment variables in order to implement Docker deployment with docker-compose.

### CODE REVIEW

**Comment by WorldObservationLog** [comment]
need to install Chinese font. the default font is Noto Sans.
`apt install fonts-noto-cjk`

**Comment by FallenDoll** [comment]
reopen

**Comment by FallenDoll** [comment]
reopen

**Comment by FallenDoll** [comment]
Thank you for your feedback and suggestions. I'm working on these issues and plan to submit another Pull Request once the work is ready. I will address these issues in the new PR, so please stay tuned.

**Review by WorldObservationLog — APPROVED** [review]
